crypto/rsa/rsa_sign.c

```diff
@@ -204,19 +335,19 @@ int int_rsa_verify(int type, const unsigned char *m, unsigned int m_len,
             }
 
             m_len = EVP_MD_size(md);
```

Comment:
coverity should be complaining about this line.. EVP_MD_size() returns a int.

Reply:
Is it or isn't it? I didn't change that line or the type of m_len.

Reply:
yep I know you didnt change it :) But it is wrong..
It needs to assign it to an int 
e.g.
int len;
len = EVP_MD_size(md); 
if (len <= 0) 
    goto error;
m_len = (cast)i;

Reply:
Yeah, okay. Done.